### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by ChanikaRuchini**
Related Issue 
- https://github.com/asgardeo/asgardeo-tomcat-oidc-agent/issues/11
- https://github.com/wso2-enterprise/asgardeo-product/issues/9654

### CODE REVIEW

**Comment by chamathns** [comment]
let's add the new configuration https://github.com/asgardeo/asgardeo-tomcat-oidc-agent/blob/master/io.asgardeo.tomcat.oidc.sample/src/main/resources/configuration-catalog.md as well

**Comment by ChanikaRuchini** [comment]
> let's add the new configuration https://github.com/asgardeo/asgardeo-tomcat-oidc-agent/blob/master/io.asgardeo.tomcat.oidc.sample/src/main/resources/configuration-catalog.md as well

Added

File: io.asgardeo.tomcat.oidc.sample/src/main/resources/configuration-catalog.md [hunk 1]
```diff
@@ -177,3 +177,17 @@ indexPage=/oidc-sample-app/index.html
 ```
 errorPage=/error.jsp
 ```
+
+### Home Page
+
+**Property Name:** `homePage`
+
+**Description:** This may denote the URI for the home page of the webapp. In case there are multiple pages in the 
+application and the user should be always redirected to a specific page after successful authentication this parameter can be configured.
```
Comment (chamathns) [inline io.asgardeo.tomcat.oidc.sample/src/main/resources/configuration-catalog.md]: Let's rephrase this a bit. Suggestion: this parameter can be configured to specify the page that the user would always be redirected to, after successful authentication.

Comment (ChanikaRuchini) [reply io.asgardeo.tomcat.oidc.sample/src/main/resources/configuration-catalog.md]: Fixed

**Review by chamathns — APPROVED** [review]

**Pull request merged**